packages/math/init.lua

```diff
@@ -67,16 +68,28 @@ function package:registerCommands ()
     xpcall(function()
       mbox = self:ConvertMathML(content, mbox)
     end, function(err) print(err); print(debug.traceback()) end)
-    self:handleMath(mbox, mode)
+    self:handleMath(mbox, mode, nil)
   end)
 
   self:registerCommand("math", function (options, content)
     local mode = (options and options.mode) and options.mode or "text"
     local mbox
+    local counter = (options and options.numbered) and "equation"
```

Comment:
`options.numbered` -> `SU.boolean(options.numbered, false)`

(Coerce to boolean, defaults to false, errors if not a boolean.)

Reply:
Also I don't think you need to check "option" (IIRC, it is set to `{}` if no options are passed), so this might suffice:
```
local counter = SU.boolean(options.numbered, false) and "equation"
```